lua/nvim-treesitter/configs.lua

```diff
@@ -166,6 +166,7 @@ local config = {
     },
     is_supported = function() return true end
   },
+  ensure_installed = {},
```

Comment:
config keys are actually used by `:TSModuleInfo`, maybe we should rename it to 'modules', i'll do that in a PR. So you can remove ensure_installed from here.